dev/tasks/tasks.yml

```diff
@@ -1040,6 +1052,7 @@ tasks:
       flags: '-e ARROW_SOURCE_HOME="/arrow" -e FORCE_BUNDLED_BUILD=TRUE -e LIBARROW_BUILD=TRUE'
       image: ubuntu-r
 
+  # This one has -flto=auto
```

Comment:
I believe `test-ubuntu-r-21.04` also has `-flto=auto` As well

Reply:
You are correct, will add a comment there too.